### ISSUE TITLE
Huge boss damage

### ISSUE THREAD

**Issue by dbueller** [HabitRPG/habitica#2562]
My team was wiped out by the gryphon doing an inexplicably huge amount of damage.  I'm a little sad about my loss of 2000 gold and an armor piece.

It was damage from only 1 undone daily.

Nimsalia attacks Fiery Gryphon for 0.0 damage, Fiery Gryphon attacks party for 64.6 damage. 

Anyone else experience that?

**Comment by lefnire** [comment]
Sure Nimsalia didn't miss all her dailies? That's what the 0.0 says to me, and we fixed 1st-day-no-damage bug. FWIW, Gryphon has 1.5 STR - meaning he does 1.5x the damage that missed dailies deal, but to the whole party.

**Comment by dbueller** [comment]
She told me that she clicked them all but one. Could have been disconnected
when she clicked them? I saw some kind of error screen pop up yesterday
evening.

**Comment by dbueller** [comment]
I was thinking about this some more.  It was the first time that the boss did damage to the group, and there was at least 1 cron between the offer of the challenge to my group and the begin of the challenge.  Is it possible that double (or triple) damage was dealt due to the lag between the "start" and when I clicked "begin?."  I would believe that 3x or maybe 4.5x damage was dealt.  I tried looking through the code, but it is a lot longer than the last time I looked at it, and I wasn't able to track it down.  Anyone with a running local copy want to test it out for me?
